src/main/resources/application.properties

```
@@ -1,7 +1,7 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/book_store?serverTimeZone=UTC
-spring.datasource.username=
-spring.datasource.password=
+spring.datasource.username=root
+spring.datasource.password=DimaSlava
```

try to set up Environment variables in the IDEA run configuration to not push anything by mistake anymore, we did this on QnA on friday afternoon, you can check the recording of it
```suggestion
spring.datasource.username=${DB_USERNAME}
spring.datasource.password=${DB_PASSWORD}
```